app/src/main/java/com/example/imdb/ProfileFragment.kt

```diff
@@ -0,0 +1,58 @@
+package com.example.imdb
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.navigation.fragment.findNavController
+import com.example.imdb.databinding.FragmentProfileBinding
+
+class ProfileFragment : Fragment() {
+    private var cardsAdapter: CardsAdapter? = null
+    private var _binding: FragmentProfileBinding? = null
+    private val binding get() = _binding!!
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        arguments?.let {
+        }
+    }
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View {
+        _binding = FragmentProfileBinding.inflate(inflater, container, false)
+        return binding.root
+    }
+
+    override fun onResume() {
+        super.onResume()
+        val userName = (activity as AfterLoginActivity).userName
+        binding.itemNameUser.text = userName
+        showCard(cardsList())
+
```

Comment:
avoid this blank spaces at the end of a function or class

Reply:
I'll implement it, thanks